SpatialGDK/Source/SpatialGDKEditor/Private/SpatialTestSettings.cpp

```diff
@@ -17,7 +20,7 @@ const FString FSpatialTestSettings::OverrideSettingsFileExtension = TEXT(".ini")
 const FString FSpatialTestSettings::OverrideSettingsFileDirectoryName = TEXT("MapSettingsOverrides");
 const FString FSpatialTestSettings::OverrideSettingsFilePrefix = TEXT("TestOverrides");
 const FString FSpatialTestSettings::OverrideSettingsBaseFilename =
-	FPaths::ConvertRelativePathToFull(FPaths::ProjectConfigDir() / OverrideSettingsFileDirectoryName) / OverrideSettingsFilePrefix;
+	FPaths::ConvertRelativePathToFull(FPaths::ProjectConfigDir() / OverrideSettingsFileDirectoryName);
```

Comment:
Does this also work with the generated configs? I think we reuse these variables, but not sure if we use this one

Reply:
Yes, the generated configs have their own base directory. I tested by running all automated tests and they all pass.
```
const FString FSpatialTestSettings::GeneratedOverrideSettingsDirectory =
	FPaths::ConvertRelativePathToFull(FPaths::ProjectIntermediateDir() / TEXT("Config/") / OverrideSettingsFileDirectoryName);
const FString FSpatialTestSettings::GeneratedOverrideSettingsBaseFilename = GeneratedOverrideSettingsDirectory / OverrideSettingsFilePrefix;
...
FString GeneratedMapOverridesFilename =
		GeneratedOverrideSettingsBaseFilename + FPackageName::GetShortName(MapName) + (OverrideSettingsFileExtension);
```